src/applications/model/gaming-streaming-server.cc

```diff
@@ -252,13 +262,13 @@ void
 GamingStreamingServer::ScheduleNextTx (int idx)
 {
   NS_LOG_FUNCTION (this << idx);
-  double nextTx = -1;
-  while (nextTx < 0)
+  Time nextTx = Seconds (-1);
+  do
     {
-      nextTx = (m_trafficStreams[idx].interArrivalTimesVariable->GetValue ());
-    }
-  m_trafficStreams[idx].sendEvent = Simulator::Schedule (Seconds (nextTx)/1e3,
-                                                         &GamingStreamingServer::Send, this, idx);
+      nextTx = Seconds(m_trafficStreams[idx].interArrivalTimesVariable->GetValue ())/1e3;
```

Comment:
Use MilliSeconds (). More elegant

Reply:
The problem with MilliSeond is that it doesn't accept double and cast it to int.

Reply:
So the interarrival times always be in an integer between 0 to 10.

Reply:
OK, then let's divide by 1e3 inside Seconds ()